React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -591,14 +591,21 @@ - (void)invalidateLayer
 
     RCTBorderColors borderColors = RCTCreateRCTBorderColorsFromBorderColors(borderMetrics.borderColors);
 
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+  CGFloat scaleFactor = self.window.backingScaleFactor;
```

Comment:
I like that the scale factor comes from the view here, instead of the main window or some other global =)

Reply:
I did some research and I thought / hoped we could just always use
[NSScreen mainScreen]. backingScaleFactor
... but it seems it is not always == to self.window.backingScaleFactor

Otherwise we could avoid change the signature of these functions as we could simply internally get the current:
[NSScreen mainScreen]. backingScaleFactor
value

Reply:
Wouldn't that not be good for multi window scenarios?